hooks/api_server.py

```diff
@@ -123,4 +103,3 @@ def __init__(self, *args, **kwargs):
         self.add_route("/health", Health())
         self.add_route(_API_PREFIX + "/{scope}/fb/hooks", FacebookHooks())
         self.add_route(_API_PREFIX + "/proxy", CorsProxy())
-        self.add_sink(CorsProxy(), _API_PREFIX + "/proxy/")  # FIXME: this is obsolete and should be removed once proxy with "?url=..." will be used everywhere
```

Comment:
@Joozty You can also remove lines 60-62 because method `__call__` is no longer needed. It was used by sink, but now only method `on_get` is required.

Reply:
@Joozty Along the description of call method, so 60-66 :-)